pyproject.toml

```
@@ -0,0 +1,9 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "setuptools_scm[toml]>=6.4",
+    "wheel",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
```

https://github.com/aio-libs/aiomysql/pull/734/files#r812397051 can be implemented via
```suggestion
[tool.setuptools_scm]
write_to = "aiomysql/_scm_version.py"
```
and `aiomysql/.gitignore` containing
```gitignore
/_scm_version.py
```
And to make up for missing typing, a pyi file could be created, like https://github.com/sphinx-contrib/sphinxcontrib-towncrier/blob/master/src/sphinxcontrib/towncrier/_scm_version.pyi.

---

implemented in #748